xml/tuning_tuned.xml

```diff
@@ -0,0 +1,2519 @@
+                <callout arearefs="include-balanced">
+                  <para>
+                    Include the <literal>balanced</literal> profile as a base, which is suitable
+                    for general purpose use.
+                  </para>
+                </callout>
+                <callout arearefs="cpu-settings-override">
+                  <para>
+                    Overrides CPU parameters such as <literal>governor</literal> and
+                    <literal>energy-perf-bias</literal> to <literal>powersave</literal>.
+                  </para>
+                </callout>
+                <callout arearefs="inherit-disk-balanced">
+                  <para>
+                    Inherit disk settings from the <literal>balanced</literal> profile.
+                  </para>
+                </callout>
+                <callout arearefs="reduce-vm-swappiness">
+                  <para>
+                    Add a sysctl parameter to reduce swappiness, which can be beneficial for both
+                    performance and power consumption in certain scenarios.
+                  </para>
+                </callout>
+              </calloutlist>
+            </step>
+            <step>
+              <para>
+                Save the configuration file and exit the text editor.
+              </para>
+            </step>
+            <step>
+              <para>
+                Apply the new custom profile after merging certain parameters from
+                <literal>balanced</literal> and <literal>powersave</literal> profiles.
+              </para>
+<screen>&prompt.sudo;<command>tuned-adm profile <replaceable>CUSTOM_PROFILE_NAME</replaceable></command></screen>
+            </step>
+          </procedure>
+        </example>
+      </sect3>
+    </sect2>
+  </sect1>
+  <sect1 xml:id="sec-tuning-tuned-plugins">
+    <title>&tunedapp; plug-ins</title>
+
+    <para>
+      &tunedapp; plug-ins are modular components that extend the functionality of the &tunedapp;
+      system daemon. These plug-ins allow system administrators to create custom optimization
+      profiles tailored to specific workloads or hardware configurations. By leveraging several
+      system metrics and user-defined parameters, &tunedapp; plug-ins can dynamically adjust kernel
+      settings, CPU frequencies, disk I/O schedulers, and other low-level system parameters to
+      achieve optimal performance and energy efficiency. While the core <package>tuned</package>
+      package includes several preconfigured profiles, you can extend it through custom plug-ins
+      for fine-grained control over the system.
+    </para>
+
+    <para>
+      The general syntax for including plug-ins in the &tunedapp; profile configuration files are
+      as follows:
+    </para>
+
+<screen>
+...
+[<replaceable>PLUGIN-NAME</replaceable>]
+<replaceable>PARAMETER</replaceable>=<replaceable>VALUE</replaceable>
+<replaceable>PARAMETER</replaceable>=<replaceable>VALUE</replaceable>
```

Comment:
Maybe make sure they are different values?

```suggestion
<replaceable>PARAMETER1</replaceable>=<replaceable>VALUE1</replaceable>
<replaceable>PARAMETER2</replaceable>=<replaceable>VALUE2</replaceable>
```

Reply:
Good suggestion. Added an underscore for readability.